mypy/test/data.py

```diff
@@ -556,6 +556,9 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    def __init__(self, *, update_data: bool = False) -> None:
```

Comment:
Yeah, that explanation of how they are called makes sense.

I still think that now you're adding `__init__` to `DataSuite`, all subclasses of it that also define `__init__` should be modified to call `super().__init__(update_data=update_data)`. Otherwise my spider-sense about inheritance is unhappy.